clip-recap-pipeline/src/index.ts

```
@@ -1727,27 +1944,19 @@ export default {
-        const apiKey = request.headers.get('x-api-key');
+        // Step 3: Authorization check - require HMAC authentication
+        const { verifyHmacSignature, createUnauthorizedResponse } = await import('./utils/auth.js');
         
-        // Check for valid authorization using centralized verification
-        const { verifyAdminAuthWithApiKey } = await import('./utils/auth.js');
-        const isAuthorized = verifyAdminAuthWithApiKey(request, env);
+        // Get request body for HMAC verification
+        const body = await request.text();
         
-        if (!isAuthorized) {
+        if (!(await verifyHmacSignature(request, env, body))) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'body'. Consider to delete it or rename it.

'body' is defined here:


(lint/suspicious/noRedeclare)